app/controllers/reservations_controller.rb

```diff
@@ -8,6 +8,15 @@ def create
     end
   end
 
+  def all_reservations
+    if admin?
+      @reservations = Reservation.all.order('created_at DESC')
```

Comment:
[Optional] It can be better to order our reservation by id, instead of created_at which is the same thing, but using the primary key could be somewhat faster.

Reply:
@aimalamiri great advice, I will implement it before merging.